src/exercise_1/index.js

```diff
@@ -0,0 +1,20 @@
+//change font family "Arial, sans-serif"
+document.body.style.fontFamily = 'Arial, sans-serif';
+
+//input information
+document.getElementById('nickname').innerHTML = 'Tai Le';
+document.getElementById('favorites').innerHTML = 'Football';
+document.getElementById('hometown').innerHTML = 'Da Nang';
+
+// change li with color red
+const items = document.getElementsByTagName('li');
+for (let i = 0; i < items.length; i++) {
```

Comment:
@taile0310 Please using a meaning variable instead of magic name as "I", "j". You can use "index"